cmd/witness/cmd/keyloader.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testifysec/witness/cmd/witness/options"
+	"github.com/testifysec/witness/pkg/cryptoutil"
+	"github.com/testifysec/witness/pkg/signer/file"
+	"github.com/testifysec/witness/pkg/signer/fulcio"
+	"github.com/testifysec/witness/pkg/signer/spiffe"
+)
+
+func getSigners(ko options.KeyOptions) ([]cryptoutil.Signer, []error) {
+	signers := []cryptoutil.Signer{}
+	errors := []error{}
+
+	ctx := context.Background()
```

Comment:
is there a reason to shuffle this to inlined vs a parameter and renaming this function?  not really a big deal.  my idea behind having this context be a parameter was the ability to add timeouts/etc at the call site.

was trying to use the language `loadXxx` for functions that opened files/etc.

guess it doesn't matter much.

Reply:
yeah, those changes don't make sense.  I'll change back to load and add the ctx as a parameter.